packages/visual-dev/src/components/Badge/Styles.ts

```diff
@@ -0,0 +1,66 @@
+import { css } from 'styled-components'
+
+export const statusColors = {
+    info: {
+      background: "#E2E2E2",
+      border: "#E2E2E2",
+      color: "#575757",
+    },
+    success: {
+      background: "#E4FCE3",
+      border: "#E4FCE3",
+      color: "#232323",
+    },
+    active: {
+      background: "#57AC59",
+      border: "#57AC59",
+      color: "#fff",
+    },
+    critical: {
+      background: "#FE6666",
+      border: "#FE6666",
+      color: "#fff",
+    },
+    warning: {
+      background: "#F1C359",
+      border: "#F1C359",
+      color: "#232323",
+    }
+  };
+  
+export const base = css`
+    border-radius: 50px;
+    padding: 4px 20px;
+    font-weight: 500;
+    font-size: 14px;
+    font-style: normal;
+    font-family: "Helvetica Neue", Helvetica, sans-serif;
+    line-height: 20px;
+`
+
```

Comment:
@Locrian24 For reference, the `statusColors` object was a solution to extract the color property for the icon (line 69 of `packages/visual-dev/src/components/Badges.tsx`). This seemed preferable to using string manipulation to pull out the color from the css string. I agree though @noahwc, it's a little clunky to duplicate all the variant definitions as strings

Reply:
I think we should avoid leaking the styling into the component file itself. We could try out `inherit` on the color prop of the icon, or using child selectors in the variant css to change the icon color instead of needing to pull the color values into the actual component file. Might be something to play around with during some pair programming @ashleyvansp 👀